test/integration/puppetlabs/puppetserver/bootstrap_testutils.clj

```diff
@@ -33,24 +33,37 @@
   [& args]
   (str (apply fs/file master-conf-dir "ssl" args)))
 
-(defmacro with-puppetserver-running
-  [app config-overrides & body]
+(defn load-dev-config-with-overrides
+  [overrides]
   (let [tmp-conf (ks/temp-file "puppet-server" ".conf")]
     (fs/copy dev-config-file tmp-conf)
-    (let [config (-> (tk-config/load-config (.getPath tmp-conf))
-                     (assoc-in [:global :logging-config] logging-test-conf-file)
-                     (assoc-in [:jruby-puppet :master-conf-dir] master-conf-dir)
-                     (assoc-in [:jruby-puppet :master-code-dir] master-code-dir)
-                     (assoc-in [:jruby-puppet :master-var-dir] master-var-dir)
-                     (assoc-in [:jruby-puppet :master-run-dir] master-run-dir)
-                     (assoc-in [:jruby-puppet :master-log-dir] master-log-dir)
-                     (ks/deep-merge config-overrides))]
-      `(let [services# (tk-bootstrap/parse-bootstrap-config! ~dev-bootstrap-file)]
-         (tk-testutils/with-app-with-config
-           ~app
-           services#
-           ~config
-           ~@body)))))
+    (-> (tk-config/load-config (.getPath tmp-conf))
+        (assoc-in [:global :logging-config] logging-test-conf-file)
+        (assoc-in [:jruby-puppet :master-conf-dir] master-conf-dir)
+        (assoc-in [:jruby-puppet :master-code-dir] master-code-dir)
+        (assoc-in [:jruby-puppet :master-var-dir] master-var-dir)
+        (assoc-in [:jruby-puppet :master-run-dir] master-run-dir)
+        (assoc-in [:jruby-puppet :master-log-dir] master-log-dir)
+        (ks/deep-merge overrides))))
+
+(defmacro with-puppetserver-running-with-services
```

Comment:
I probably could have refactored this to be a bit DRY'er, but really didn't want get caught up futzing with macros